ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -433,7 +433,10 @@ export const ProfilePage = fp.flow(
 
       const makeProfileInput = ({title, valueKey, isLong = false, ...props}) => {
         let errorText = profile && errors && errors[valueKey];
-        if (valueKey && Array.isArray(valueKey) && valueKey.length > 1) {
+        if (valueKey && !Array.isArray(valueKey)) {
+          valueKey = [valueKey];
+        }
+        if (valueKey && valueKey.length > 1) {
```

Comment:
The `profile-page.spec.tsx` was failing consistently after versions upgrade. @CH helped me and figured out it was caused by a bug here.